addons/point_of_sale/static/tests/tours/utils/number_popup_util.js

```diff
@@ -1,9 +1,12 @@
 import * as Numpad from "@point_of_sale/../tests/tours/utils/numpad_util";
 
 export function enterValue(keys) {
-    return Numpad.enterValue(keys).map((step) => ({
-        ...step,
-    }));
+    return Numpad.enterValue(keys).map((step) => {
+        step.trigger = `.modal ${step.trigger}`;
+        return {
+            ...step,
+        };
+    });
```

Comment:
This has no incidence in this case since `enterValue` returns fresh objects every time but you may favor the spread & override notation to not alter the original object (also, spreading the `step` like you did does not add anything)
```suggestion
    return Numpad.enterValue(keys).map((step) => ({
        ...step,
        trigger: `.modal ${step.trigger}`,
    }));
```